CodeStarterCamp_Week1/main.swift

```diff
@@ -13,73 +13,53 @@ func drawPeperoBody(body: String, topping: String) {
     print(topping + body + topping)
 }
 
-func drawPeperoBar(lenght: Int, topping: String, body: String) {
+func drawPeperoBodyAndLength(length: Int, topping: String, body: String) {
     if body == "***" {
-        for _ in 1...lenght {
+        for _ in 1...length {
             drawPeperoBody(body: body, topping: topping)
         }
     }
     else {
-        for i in 1...lenght {
-            if (i % 2) != 0{
+        for oddNumber in 1...length {
+            if (oddNumber % 2) != 0 {
```

Comment:
for 문에서 넘겨받는 모든 숫자가 홀수(oddNumber)는 아니지 않나요?! 👀 
```suggestion
        for index in 1...length {
            let isOddNumber = (index % 2) != 0
            if isOddNumber {
```
이렇게 조건에 의미를 부여하면, 코드를 읽는 입장에서 if 문을 어떤 의도로 넣었는지 좀 더 쉽게 이해할 수 있어요! 🙂

Reply:
다음에는 참고하여 코드를 작성하겠습니다!!